editor/src/clj/editor/defold_project.clj

```diff
@@ -1509,6 +1510,7 @@
   (input collision-group-nodes g/Any :array :substitute gu/array-subst-remove-errors)
   (input build-settings g/Any)
   (input breakpoints Breakpoints :array :substitute gu/array-subst-remove-errors)
+  (input proj-path->meta-info g/Any :array :substitute gu/array-subst-remove-errors)
```

Comment:
The input name suggests this takes maps. Maybe it would make sense to rename it to something like `proj-path+meta-info-pairs`?